confd/confd_test.go

```diff
@@ -0,0 +1,74 @@
+package confd_test
```

Comment:
Hi, there. There are multiple possibilities in golang when testing the same method for different cases. It is recommended to group all test cases in the method under test. Take a look at some of the test cases in the cesapp for better understanding: https://github.com/cloudogu/cesapp/blob/develop/containers/containerProviderRegistry_test.go

We should group the test cases by the `GetStringValue`-method and the `GetAttributes`-method.

Reply:
I have grouped the test cases by the `GetStringValue`-method and `GetAttributes`-method.